common/djangoapps/student/tests/test_bulk_email_settings.py

```diff
@@ -51,19 +51,21 @@ def setUp(self):
             name=self.course.display_name.replace(' ', '_'),
         )
 
-    @patch.dict(settings.FEATURES, {'ENABLE_INSTRUCTOR_EMAIL': True, 'REQUIRE_COURSE_EMAIL_AUTH': False})
+    @patch('bulk_email.models.BulkEmailFlag.is_enabled', Mock(return_value=True))
```

Comment:
Would it be possible to just turn the `BulkEmailFlag` in the `setUp` instead of having to mock it out this way? We used to have to patch the dict because of the way settings work, but with database configs, it's probably easier to just turn it on for the whole class.